doc/README.md

```diff
@@ -1,7 +1,29 @@
 # Installation for OSX Development
-- Install gcc on your Mac via running ```gcc``` on the terminal
-- This will prompt you to install XCode
+TODO(raj): Write a script to do all this
+
+- Check if gcc is installed on your Mac via running ```gcc``` on the terminal
+- If it is not then this will prompt you to install XCode
 - Install XCode
 - Verify gcc was installed by running ```gcc```
+- Install git (search for it online)
+- Run ```git clone https://github.com/supragma/supragma.git```
+- Run inserting your username and email (instead of Raj's):
+```
+  git config --global user.name "Navraj Chohan"

```

Comment:
I would just link to this and tell them to follow it. https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration